edgemesh/pkg/plugin/registry/registry.go

```diff
@@ -75,28 +70,18 @@ func (esd *EdgeServiceDiscovery) FindMicroServiceInstances(consumerID, microServ
 
 	// gen
 	var microServiceInstances []*registry.MicroServiceInstance
-	var hostPort int32
-	// all pods share the same hostport, get from pods[0]
-	if pods[0].Spec.HostNetwork {
-		// host network
-		hostPort = int32(targetPort)
-	} else {
-		// container network
-		for _, container := range pods[0].Spec.Containers {
-			for _, port := range container.Ports {
-				if port.ContainerPort == int32(targetPort) {
-					hostPort = port.HostPort
-				}
-			}
-		}
```

Comment:
I'd like to see the reason behind those change, either in the commit message or an issue filed somewhere.

Reply:
See #1856